Dockerfile.gccgo

```diff
@@ -41,8 +41,7 @@ RUN cd /usr/local/lvm2 \
 	&& make install_device-mapper
 # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
 
-# install seccomp
-# TODO: switch to libseccomp-dev since dockerinit is gone
+# install seccomp: the version shipped in jessie is too old
```

Comment:
yeah :(

On Tue, Feb 2, 2016 at 10:06 AM, Tianon Gravi <notifications@github.com>
wrote:

> In Dockerfile.gccgo
> <https://github.com/docker/docker/pull/19927#discussion_r51608073>:
>
> > @@ -41,8 +41,7 @@ RUN cd /usr/local/lvm2 \
> >  	&& make install_device-mapper
> >  # see https://git.fedorahosted.org/cgit/lvm2.git/tree/INSTALL
> >
> > -# install seccomp
> > -# TODO: switch to libseccomp-dev since dockerinit is gone
> > +# install seccomp: the version shipped in jessie is too old
>
> aww for reals?? jessie's too old too???
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/19927/files#r51608073>.
>



-- 


Jessie Frazelle
4096R / D4C4 DD60 0D66 F65A 8EFC  511E 18F3 685C 0022 BFF3
pgp.mit.edu <http://pgp.mit.edu/pks/lookup?op=get&search=0x18F3685C0022BFF3>